monitoring/grafana/dashboards/ceph-cluster.json

```diff
@@ -1245,7 +1245,7 @@
       "30d"
     ]
   },
-  "timezone": "browser",
+  "timezone": "",
```

Comment:
In order to enforce this, I'd suggest that you add an assertion in the `ci/check_grafana_uids.py` script (and probably we can rename it that to `check_grafana_dashboards` or something) to ensure this cannot be (unintentionally) changed.